### PR TITLE
updates vonage to to the new api

### PR DESCRIPTION
**Pull request by ptrckbp**

### CODE REVIEW

File: docs/messaging-channels/broker-integrations/vonage/local.md [hunk 1]
```diff
@@ -10,15 +10,16 @@
...
   - Click **Generate new application**
 
 ### Get your API credentials
 
 1. Go to your [Account Settings](https://dashboard.nexmo.com/settings).
 2. Scroll down and copy your API key, API secret and Signature secret from the LIVE credentials section.
-3. Go to your applications and select your newly created application (https://dashboard.nexmo.com/applications/).
-4. Copy the Application ID.
+3. In this page, under `SMS Settings`, make sure SMS Api is selected. 
+4. Under `Signature method method`, select MD5 HASH.
```
Comment (laurentlp) [inline docs/messaging-channels/broker-integrations/vonage/local.md:20]:
```suggestion
4. Under `Signature method`, select MD5 HASH.
```

File: docs/messaging-channels/broker-integrations/vonage/local.md [hunk 2]
```diff
@@ -10,15 +10,16 @@
...
-  - Under `Authentication`, click **Generate public and private key** (this will download a file called `private.key`. You will need this file later on.)
   - Click **Generate new application**
 
 ### Get your API credentials
 
 1. Go to your [Account Settings](https://dashboard.nexmo.com/settings).
 2. Scroll down and copy your API key, API secret and Signature secret from the LIVE credentials section.
-3. Go to your applications and select your newly created application (https://dashboard.nexmo.com/applications/).
-4. Copy the Application ID.
+3. In this page, under `SMS Settings`, make sure SMS Api is selected.
```
Comment (laurentlp) [inline docs/messaging-channels/broker-integrations/vonage/local.md:19]:
```suggestion
3. On this page, under `SMS Settings`, make sure SMS API is selected. 
```

File: docs/messaging-channels/broker-integrations/vonage/local.md [hunk 3]
```diff
@@ -10,15 +10,16 @@
...
 
 ### Get your API credentials
 
 1. Go to your [Account Settings](https://dashboard.nexmo.com/settings).
 2. Scroll down and copy your API key, API secret and Signature secret from the LIVE credentials section.
-3. Go to your applications and select your newly created application (https://dashboard.nexmo.com/applications/).
-4. Copy the Application ID.
+3. In this page, under `SMS Settings`, make sure SMS Api is selected. 
+4. Under `Signature method method`, select MD5 HASH.
+5. Go to your applications and select your newly created application (https://dashboard.nexmo.com/applications/).
```
Comment (laurentlp) [inline docs/messaging-channels/broker-integrations/vonage/local.md:21]: Is "your applications" a button on some specific place you can navigate to? If so, make sure to put emphasis on it. Something like: Go to `your applications` and ...

File: docs/messaging-channels/broker-integrations/vonage/local.md [hunk 4]
```diff
@@ -28,10 +29,6 @@
 - `apiKey`: paste your API key;
 - `apiSecret`: paste your API Secret;
 - `signatureSecret`: paste your Signature secret;
-- `applicationId`: paste your Application ID;
```
Comment (laurentlp) [inline docs/messaging-channels/broker-integrations/vonage/local.md:31]: Wh do we need to copy the Application ID if we don't have to provide it in the settings?

File: docs/messaging-channels/broker-integrations/vonage/local.md [hunk 5]
```diff
@@ -68,9 +63,9 @@ When wanting to test the integration with Vonage and WhatsApp, you need to enabl
 1. Click the channel you want to setup and follow the instructions detailed on the page.
 1. Under `Webhooks`, type the following URLs:
 
-   _Inbound:_ `<EXTERNAL_URL>/api/v1/messaging/webhooks/<YOUR_BOT_ID>/vonage/inbound`
+   _Inbound:_ `<EXTERNAL_URL>/api/v1/messaging/webhooks/v1/<YOUR_BOT_ID>/vonage`
 
-   _Status:_ `<EXTERNAL_URL>/api/v1/messaging/webhooks/<YOUR_BOT_ID>/vonage/status`
+   _Status:_ `<EXTERNAL_URL>/api/v1/messaging/webhooks/v1/<YOUR_BOT_ID>/vonage`
```
Comment (laurentlp) [inline docs/messaging-channels/broker-integrations/vonage/local.md:68]: Don't forget to add the `"version": "1.0.0"` entry in the config template. `v1` Webhooks only works when using version 1.0.0 of a given channel.

**Review by rohankokkulabp — APPROVED** [review]

**Pull request merged**